client/client.go

```diff
@@ -140,7 +140,7 @@ func NewTarget(targetName string, targetPath string) (*Target, error) {
 		return nil, err
 	}
 
-	meta, err := data.NewFileMeta(bytes.NewBuffer(b))
+	meta, err := data.NewFileMeta(bytes.NewBuffer(b), "sha256", "sha512")
```

Comment:
Related to https://github.com/docker/notary/pull/593/files#r54474516 - if we could replace this with a constant that knows which hashes are supported, that would be great.  :)

But if you didn't want to have a single collection though, it'd still be nice to use the awesome constants that you added - `notary.SHA256` and `notary.SHA512`